x-pack/plugins/features/server/ui_capabilities_for_features.test.ts

```diff
@@ -27,14 +43,14 @@ describe('populateUICapabilities', () => {
   it('handles features with no registered capabilities', () => {
     expect(
       uiCapabilitiesForFeatures([
-        {
+        new Feature({
           id: 'newFeature',
           name: 'my new feature',
           app: ['bar-app'],
-          privileges: {
+          privileges: ({
             ...createFeaturePrivilege('all'),
-          },
-        },
+          } as unknown) as FeatureConfig['privileges'],
```

Comment:
(did not check reasons of mismatch, but) Can we adapt `createFeaturePrivilege` to avoid this `as unknown) as FeatureConfig['privileges']` ?

Reply:
Ah, this is complaining because the `privileges` object, when defined, expects both a `read` and an `all` property, and these tests, for the sake of brevity, do not include both. I can likely clean this up though, let me take a stab at it.